charts/newrelic-pixie/values.yaml

```diff
@@ -23,6 +23,12 @@
 # The key in the customSecretApiKeyName secret that contains the Pixie API key
 # customSecretApiKeyKey:
 
+clusterRegistrationWaitImage:
+  repository: gcr.io/pixie-oss/pixie-dev-public/curl
+  tag: "1.0"
+  pullPolicy: IfNotPresent
+  pullSecrets: []
```

Comment:
where do the `clusterRegistrationWaitImage.pullSecrets` get used?

Reply:
Removed, was stale from the other image spec.